src/components/atoms/Ranges/DefaultRange/index.ts

```diff
@@ -0,0 +1,3 @@
+import Range from './Range';
+
+export default Range;
```

Comment:
It could be in the root of ranges

```suggestion
import Range from './Range';
import LevelRange from './LevelRange';

export {Range, LevelRange};
```

Reply:
ok